platform/README.md

```diff
@@ -0,0 +1,73 @@
+Guice Platform Bill of Materials (BOM)
+============================
+This BOM is an extension of of the Guice BOM. It appends all dependencies needed to run Guice and its extensions.
```

Comment:
The only difference I can see is the Guice Platform adds further version properties for transitive dependencies of components already in the Guice BOM. So this extra pom is only useful if you want to override these transitive dependency versions using properties - but really you should be using `<dependencyManagement>` to do this because a) it's the standard way to override versions and b) it works for all dependencies, whereas the version properties approach only works for projects/BOMs which follow the same property pattern